File: Projeto_7/src/Transacao.cpp
#include "Transacao.h"

Transacao::Transacao(int id, double valor, std::string tipo) 
    : id(id), valor(valor), tipo(tipo) {}

int Transacao::getId() const { return id; }
double Transacao::getValor() const { return valor; }
std::string Transacao::getTipo() const { return tipo; }

File: Projeto_7/src/Transacao.py
class Transacao:
    def __init__(self, id_transacao: int, valor: float, tipo: str):
        self.__id = id_transacao
        self.__valor = valor
        self.__tipo = tipo

    def get_valor(self) -> float:
        return self.__valor

    def get_tipo(self) -> str:
        return self.__tipo

    def __str__(self) -> str:
        return f"Transação [ID: {self.__id} | Valor: R${self.__valor:.2f} | Tipo: {self.__tipo}]"